content/docs/securitypolicy/_index.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Security Policy"
+linkTitle: "Security Policy"
+weight: 13
+Description: >
+  Dell Container Storage Modules (CSM) Security Policy
+---
+
+
+The CSM services/repositories are inspected for security vulnerabilities via [gosec](https://github.com/securego/gosec), 
+[Dell Vulnerability Policy](https://www.dell.com/support/contents/en-in/article/product-support/self-support-knowledgebase/security-antivirus/alerts-vulnerabilities/dell-vulnerability-response-policy#:~:text=To%20report%20a%20security%20vulnerability%20or%20issue%20in%20Dell.com,instructions%20to%20reproduce%20the%20issue)
+
+
+
+## Reporting a Vulnerability
+
+Please report a vulnerability by opening an Issue in [CSM](https://github.com/dell/csm/issues).
```

Comment:
Replace this with "Instructions for reporting a vulnerability can be found on the [Dell Vulnerability Response Policy](https://www.dell.com/support/contents/en-ca/article/product-support/self-support-knowledgebase/security-antivirus/alerts-vulnerabilities/dell-vulnerability-response-policy) page."